QATCH/processors/Analyze.py

```
@@ -9335,6 +9383,7 @@ def reject_outliers(data, m):
             std_viscosity = np.std(in_viscosity)
             # lin_viscosity = np.flip(lin_viscosity)
             for i in range(-len(distances), 0):
+                if initial_fill_only: break  # skip
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)